tools/crm_mon.c

```diff
@@ -386,18 +388,22 @@ fence_history_cb(const gchar *option_name, const gchar *optarg, gpointer data, G
 
     switch (rc) {
         case 3:
+            interactive_fence_level = 3;
```

Comment:
Nothing wrong with this but the simplest approach is s/rc/interactive/fence_level/ :)

Reply:
Hah, yes of course.  Tunnel vision.